tc_build/llvm.py

```diff
@@ -363,6 +363,8 @@ def configure(self):
         }
 
         slim_llvm_defines = {
+            # Tools needed by bootstrapping
+            'LLVM_DISTRIBUTION_COMPONENTS': 'clang;clang-resource-headers;lld;llvm-ar;llvm-nm;llvm-ranlib;llvm-objcopy;llvm-objdump;llvm-readelf;llvm-strip',
```

Comment:
It might help readability to have this be a list of individual items and use `';'.join(...)` for the actual assignment but I don't have a strong preference. I don't mind more lines for easier maintenance.